modules/dxp/apps/search-experiences/search-experiences-web/test/js/edit_sxp_blueprint/add_sxp_element_sidebar/index.js

```diff
@@ -12,8 +12,8 @@
 import {fireEvent, render} from '@testing-library/react';
 import React from 'react';
 
-import AddSXPElementSidebar from '../../../src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/AddSXPElementSidebar';
-import {QUERY_SXP_ELEMENTS} from '../mocks/data';
+import AddSXPElementSidebar from '../../../../src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/add_sxp_element_sidebar';
```

Comment:
Let's also add the `/index` to this path to keep the paths consistent even though it isn't necessary here.